metadata-ingestion/src/datahub/ingestion/source/identity/okta.py

```diff
@@ -140,6 +140,10 @@ class OktaConfig(StatefulIngestionConfigBase, ConfigModel):
         default=None,
         description="Okta search expression (not regex) for ingesting groups. Only one of `okta_groups_filter` and `okta_groups_search` can be set. See (https://developer.okta.com/docs/reference/api/groups/#list-groups-with-search) for more info.",
     )
+    ingest_users_for_filtered_groups: bool = Field(
```

Comment:
What if we named this: `skip_users_without_a_group` and then got rid of `has_group_filter()`

I feel like the behavior would be more clear